sdk/personalizer/ai-personalizer-rest/tests.yml

```
@@ -0,0 +1,22 @@
+# Parameterize Location to the test resources deployment script. This allows
+# specifying a different Azure region to use when manually triggering the live
+# tests pipeline. This is useful for when we want to test in different
+# environments: Prod, Canary, etc.
+parameters:
+- name: Location
+  displayName: Location
+  type: string
+  default: eastus2euap
+
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure-rest/ai-personalizer"
+      ServiceDirectory: personalizer
+      Location: "${{ parameters.Location }}"
```

For canary (euap) testing, this way is more consistent with our patterns:

```suggestion
      Clouds: Canary
```

And if you want you can also expose the parameter `Clouds` e.g. `Public,Canary`. `Location` will still work as a parameter to override the public cloud testing.

---

Removed Location parameter and added Clouds.